R/microserver.r

```diff
@@ -13,9 +13,8 @@ http_server <- function(routes) {
     params <- extract_params_from_request(req)
     query  <- extract_query_from_request(req)
     route  <- determine_route(routes, req$PATH_INFO)
-    # Provide access to the server environment.
-    environment(route) <-
-      list2env(list(.server_env = environment()), parent = environment(route))
+    if (is.microserver_response(route)) return(unclass(route))
+    environment(route) <- environment()
```

Comment:
I think we should keep the old one to remain compatible with the existing microservers. Otherwise route closures will break.